src/creature.js

```diff
@@ -1181,13 +1184,13 @@ export class Creature {
 	 *
 	 * damage :	Damage : 	Damage object
 	 *
-	 * return :	Object :	Contains damages dealed and if creature is killed or not
+	 * return :	Object :	Contains damages dealt and if creature is killed or not
 	 */
 	takeDamage(damage, o) {
 		let game = this.game;
 
 		if (this.dead) {
-			game.log('%CreatureName' + this.id + '% is already dead, aborting takeDamage call.');
+			console.info(`${this.name} (${this.id}) is already dead, aborting takeDamage call.`);
```

Comment:
I don't see a reason for this to be logged to player. I thought about making it a `console.warn()`, but I don't know if this really indicates something is wrong, just that this sanity check is doing its job.